lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py

```diff
@@ -244,7 +239,36 @@ def push_to_sqs(event_body):
         
         logger.info("Completed enqueue of message to retry queue")
     except Exception as e:
-        logger.error("Push to SQS Exception")
+        logger.error("Push to Retry SQS Exception")
         logger.error(event_body)
         logger.exception(e)        
-    
\ No newline at end of file
+        push_to_dead_letter_sqs(event_body, "push_to_retry_sqs")
+
+def push_to_dead_letter_sqs(event, source):
+    """Places unaccounted for event on dead-letter queue to be inspected"""
+
+    logger.info("Placing event on dead-letter queue")
+    logger.debug(f"Preparing for DeadLetter SQS: {event}")
+
+    try:
+        sqs = boto3.client('sqs')
+        queue_url = os.getenv('vetext_request_dead_letter_sqs_url')
+        logger.debug(f"Retrieved queue_url: {queue_url}")
```

Comment:
Move this to before the "try" block because it will not raise an exception.  Check that queue_url is not None before line 266.